src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.targets

```diff
@@ -405,4 +407,14 @@
   <Target Name="_Remove32BitLinux" Condition="Exists ('$(OutputPath)lib32\libZipSharpNative.so')" AfterTargets="CopyFilesToOutputDirectory">
     <Delete Files="$(OutputPath)lib32\libZipSharpNative.so" />
   </Target>
+
+  <Target Name="_SetAssemblyVersion"
+      DependsOnTargets="GetXAVersionInfo"
+      BeforeTargets="GetAssemblyAttributes">
+    <PropertyGroup>
+      <AssemblyVersion>$(ProductVersion).$(PackVersionCommitCount)</AssemblyVersion>
+      <FileVersion>$(AssemblyVersion)</FileVersion>
+      <InformationalVersion>$(AssemblyVersion)</InformationalVersion>
+    </PropertyGroup>
```

Comment:
I recently found you can just set `$(Version)` and it sets all three of these.

Do we also need to setup the version on our other assemblies?

Reply:
doh. Now we are gonna have to go through all the dependencies and make sure they all have versions set. Including Java.Interop/androidtools/xamarin-android-tools etc 🤦